### ISSUE TITLE
download twitch stream when stream didnt ended 

### ISSUE THREAD

**Issue by magorko** [ytdl-org/youtube-dl#30831]
- [ x] I'm asking a question
- [ x] I've looked through the README and FAQ for similar questions
- [ x] I've searched the bugtracker for similar questions including closed ones

I want to download twitch VOD but there is a problem. The VOD is not visssible in the twitch previous broadcasting so I cant just put the regular vod URL. Like this -  youtube-dl.exe -f 720p60 https://www.twitch.tv/videos/1297056505 . Whats the another similiar command for this ?

**Comment by dirkf** [comment]
I'm afraid you'll have to explain further. You surely aren't asking for a way to guess a URL that you don't see in your browser's address bar?

Obviously, the example you show works fine.

**Comment by magorko** [comment]
No. The broadcasting is no where to be found in past broadcasting section
on a channel. Probably bug of twitch. So I need to download stream without
the twitch videos / numbers .

Dňa pi 8. 4. 2022 o 5:18 dirkf ***@***.***> napísala(a):

**Comment by dirkf** [comment]
As I have no experience of Twitch, I can't really help. Maybe some more details and examples would help.

**Comment by MrDrummer** [comment]
It's a recent change to Twitch.

It's to help streamers with DMCA - by **_default_** now VODs will be set to not be published automatically. This means the channel you are trying to download from has the videos hidden by default. They need to go to their stream settings to change this. It's currently unclear if videos will still be set to be unpublished if Twitch detects DMCA content, and if Twitch still mutes DMCA audio (at least to the web player).

I'm still trying to figure out all of the details. The biggest annoyance is that the currently streamed VOD isn't published, which means the current video ID is not accessible publicly until it's published.
